subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/SmokeContinuousIntegrationTest.groovy

```diff
@@ -55,10 +56,17 @@ class SmokeContinuousIntegrationTest extends AbstractContinuousIntegrationTest {
         waitBeforeModification(markerFile)
         markerFile.text = "created"
         then:
-        buildTriggeredAndSucceeded()
-        output.contains "exists: true"
+        if (OperatingSystem.current().linux) {
+            buildTriggeredAndSucceeded()
+            output.contains "exists: true"
+        } else {
+            // TODO: We may want to support this use-case for
+            //       hierarchical watchers at some point as well.
+            noBuildTriggered()
+        }
     }
 
+    @ToBeImplemented
     def "detects changes when no snapshotting happens (#description)"() {
```

Comment:
I think it would be better to add a test for checking that this does not work for now, unless we do have plans to implement this. (IIUC, we don't.)

Reply:
I did that.